.github/workflows/smoke-tests.yml

```diff
@@ -45,12 +45,22 @@ jobs:
     - name: Python Checkstyle
       timeout-minutes: 20
       run: |
-        pylint --disable=F0401 --disable=W0511 --disable=E1101 --disable=R0801 --disable=R0903  ./src/*/*.py
-        # disabled warnings: import loading, todos, function members, duplicate code, public methods
+        pylint --disable=F0401 --disable=W0511 --disable=E1101 --disable=R0801 --disable=R0903 --disable=R0915 ./src/*/*.py
```

Comment:
not a change to this PR, but would it be possible to move this to a config in the future? would look cleaner and be easier to maintain.

Reply:
yes I think we can either make .pylintrc per python microservice or one for all

Reply:
created #136